apps/evm/lib/evm/configuration/frontier_test.ex

```diff
@@ -0,0 +1,15 @@
+defmodule EVM.Configuration.FrontierTest do
```

Comment:
Just curious, why is this called `FrontierTest` instead of `Frontier`?

Reply:
@germsvel I think these configurations will be used only in tests. Btw I can rename them

Reply:
If they're only going to be used in tests, should we put them somewhere within `test/support` or something like that? Also, what do you think of renaming them `TestFrontier` instead of `FrontierTest` since ex unit uses the ending-with-Test convention?